src/main/java/com/smartcar/sdk/data/Connection.java

```diff
@@ -0,0 +1,28 @@
+package com.smartcar.sdk.data;
+
+public class Connection {
+    private String vehicleId;
+    private String userId;
+    private String createdAt;
+
+    public String getVehicleId() {
+        return vehicleId;
+    }
+
+    public String getUserId() {
+        return userId;
+    }
+
+    public String getCreatedAt() {
```

Comment:
Using this as the reference thread, but this value and all other createdAt values, need to be changed to `connectedAt`. That's what the `processor.format` will return at the api layer.